[I 160527 14:44:17 web:1946] 200 GET / (::1) 3.07ms
[W 160527 14:44:17 web:1946] 404 GET /favicon.ico (::1) 0.86ms
[E 160527 14:44:21 web:1524] Uncaught exception GET /weixin (::1)
    HTTPServerRequest(protocol='http', host='localhost', method='GET', uri='/weixin', version='HTTP/1.1', remote_ip='::1', headers={'Connection': 'keep-alive', 'Accept': 'text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,*/*;q=0.8', 'Accept-Encoding': 'gzip, deflate, sdch', 'Upgrade-Insecure-Requests': '1', 'User-Agent': 'Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Ubuntu Chromium/50.0.2661.102 Chrome/50.0.2661.102 Safari/537.36', 'Host': 'localhost', 'Accept-Language': 'zh-CN,zh;q=0.8'})
    Traceback (most recent call last):
      File "/home/zhangyan/.local/lib/python3.5/site-packages/tornado/web.py", line 1443, in _execute
        result = method(*self.path_args, **self.path_kwargs)
      File "/home/zhangyan/dev/git/qilaihi.club/server/weixin/wxservice.py", line 11, in get
        signature = self.request.query_arguments['signature'][0]
    KeyError: 'signature'
[E 160527 14:44:21 web:1946] 500 GET /weixin (::1) 4.28ms
[I 160527 14:50:11 server:42] server started
